python-regression/tests/features/steps/transaction_steps.py

```diff
@@ -195,7 +364,23 @@ def issue_a_milestone(step, index, node):
     milestone_hash2 = Transaction.from_tryte_string(milestone['trytes'][1]).hash
     world.config['latestMilestone'][node] = [milestone_hash, milestone_hash2]
 
+def set_previous_transaction(node, hash):
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Redefining built-in 'hash'](https://app.codacy.com/manual/GalRogozinski/iri/pullRequest?prid=5128317)